input/i18n/en/my-account.yaml

```diff
@@ -27,3 +28,10 @@ shippingAddress:
 buttons:
   saveAddressBtn: Save changes
   cancelAddressBtn: Cancel editing
+changePassword:
+  labelTxt: Password
+  confirmLabelTxt: Confirm password
+  changePasswordTitle: Change your password
```

Comment:
If this is the title of the page, by convention we usually call it simply `title` and put it the first element of the group.
```
changePassword:
  title: Change password
  labelTxt: ...
  ...
```